nabs/tests/test_plan_stubs.py

```diff
@@ -16,6 +16,7 @@ def test_measure_average(RE, hw):
     def measure_plan(detectors):
         yield from open_run()
         ret = yield from measure_average(detectors, num=250)
+        logger.debug(ret)
```

Comment:
`ret` isn't a string, is it?

Reply:
`ret` is a `dict` here, does that cause issues?

Reply:
I thought it wasn't allowed, but apparently am wrong here as well:

```
In [1]: import logging

In [3]: logging.error({})
ERROR:root:{}

In [4]: logging.error([])
ERROR:root:[]

In [5]: logging.error(['a', 'b', 'c'])
ERROR:root:['a', 'b', 'c']
```

It is a bit weird, though, I think...

Reply:
I'll revise to build a clear log message here then, this one was helpful for debugging